kinto/plugins/history/__init__.py

```diff
@@ -0,0 +1,17 @@
+from kinto.core.events import ResourceChanged
+
+from .listener import on_resource_changed
+
+
+def includeme(config):
+    config.add_api_capability('history',
+                              description='Track changes on data.',
+                              url='https://kinto.readthedocs.io')
+
+    # Activate end-points.
+    config.scan('kinto.plugins.history.views')
+
+    # Listen to every resources (except history)
```

Comment:
Actually listeners can setup for certain kinds of objects, but not for a list of buckets/collections ids. Let's keep that for another issue. Because we might prefer have a blacklist instead etc.
From the point of view of kinto-admin, it makes it harder to implement if the tracked buckets are limited. Let's start simple.